<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="vehicle_cameras/tests/test_utils.py" name="vehicle_cameras.tests.test_utils.TestDeleteImagesAndVideosFromReports-20220725104622" skipped="0" tests="3" time="0.413" timestamp="2022-07-25T10:57:02">
	<testcase classname="vehicle_cameras.tests.test_utils.TestDeleteImagesAndVideosFromReports" file="vehicle_cameras/tests/test_utils.py" line="66" name="test_delete_images_success" time="0.142" timestamp="2022-07-25T10:57:02">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="vehicle_cameras.tests.test_utils.TestDeleteImagesAndVideosFromReports" file="vehicle_cameras/tests/test_utils.py" line="76" name="test_delete_videos_success" time="0.141" timestamp="2022-07-25T10:57:02">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="vehicle_cameras.tests.test_utils.TestDeleteImagesAndVideosFromReports" file="vehicle_cameras/tests/test_utils.py" line="87" name="test_s3_client_failure" time="0.129" timestamp="2022-07-25T10:57:02">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
